preview/how-to-use-arcus.md

```diff
@@ -0,0 +1,43 @@
+# How to use Arcus?
+Arcus' main goal is to make your application development more efficient. This includes infrastructure or boilerplate code that is almost exactly the same on every project. You can see Arcus as the middleware between your business application code and Azure components.
+
+Focusing on application code as much as possible, that' where Arcus comes in.
+
+![Code - Arcus - Azure](./media/code-arcus-azure.png)
+
+## How to use Arcus Observability
+The observability library provides the solution to a single point of truth for your logging and telemetry. The library simplifies logging by abstracting the Azure Application Insights telemetry behind the default Microsoft logging abstraction. Writing traces or informational messages becomes as simple as tracking a dependency to Application Insights.
```

Comment:
I think one of the comments I have seen is why not use the default Microsoft libraries, not sure if this document is the right place, but maybe we could include some information about this question?

Reply:
Yep, that's a valid comment, and this can definitely be place to remind people again why Arcus is a better way to do this in a company-wide setup.

Reply:
Added some answers to the Microsoft questions 😉 .

Reply:
Looks like the comments I made yesterday were still pending

Reply:
Oops, 😅 , and are the changes made a bit like you want, @fgheysels ?

Reply:
Yes, it looks great.

Your writing skills are really good.  I wonder if the sentences just flow out of your keyboard, or if it is more like my experience, where I write, rewrite, etc... :)

Reply:
Thx, @fgheysels ! Means a lot, really.
Yeah, I flows really, of course there's always editing/rewriting, but the main stuff is already there from the beginning. I also write a lot of (science)fiction in my spare time and I read a lot, that helps 😄 . But spelling is and will never by my best feature. 😅